### PR TITLE
helm-charts: Add assets.json to helm deployment

### PR DESCRIPTION
**Pull request by erika-sdf**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [x] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [x] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

Add assets.json to helm deployment
-Add asset config map and mount it to the sep server deployment
-Add documentation for assets configuration

### CODE REVIEW

**Comment by reecexlm** [comment]
love this pr. thank you! reviewing

File: helm-charts/sep-service/templates/README.md [hunk 1]
```diff
@@ -61,9 +61,75 @@ metadata:
 type: Opaque
```
Comment (reecexlm) [inline helm-charts/sep-service/templates/README.md:61]: all files in templates get processed by helm install. moving readme into templates directory broke your deployment of the sep service.   i pushed an empty commit to kick off the deployment again. here you can see the failure of helm trying to process it
https://buildmeister-v3.stellar-ops.com/job/ProductEngineering/job/stellar-anchor-platform-preview/956/

Comment (erika-sdf) [reply helm-charts/sep-service/templates/README.md]: Thanks for catching, it's been restored to its original location

File: helm-charts/sep-service/templates/configmap.yaml [hunk 2]
```diff
@@ -263,3 +261,17 @@ data:
     ORG_LOGO = "https://myorglogo.png"
     ORG_SUPPORT_EMAIL="myname@myemail.org"
     {{- end }}
+---
```
Comment (reecexlm) [inline helm-charts/sep-service/templates/configmap.yaml:264]: I think we should make this entire map optional. 
ie perhaps move {{- if .Values.assets }} to top (above the + ---)?? 
the reason is that bitso provides their config map via a custom volume mount.  If we ship it like this, we could break their implementation until they move to  providing via values file.

Comment (erika-sdf) [reply helm-charts/sep-service/templates/configmap.yaml:264]: good call, I'll make that change

**Comment by reecexlm** [comment]
is there another PR in https://github.com/stellar/kube/tree/master/local/anchor-platform-helm-values to ensure this is tested in preview?

**Comment by erika-sdf** [comment]
> is there another PR in https://github.com/stellar/kube/tree/master/local/anchor-platform-helm-values to ensure this is tested in preview?

https://github.com/stellar/kube/pull/1021

**Review by reecexlm — APPROVED** [review]
LGTM 👍 ,  may want to delete the `sep-0.3.84.tgz` helm package but feel free to merge

File: helm-charts/sep-service/README.md [hunk 3]
```diff
@@ -61,9 +61,75 @@ metadata:
 type: Opaque
 ```
 
+# Assets Configuration
```
Comment (lijamie98) [inline helm-charts/sep-service/README.md:64]: 😎 Hey, this is cool. We were discussing of converting the `assets.json` to `assets.yaml`.

**Pull request merged**